src/routes/user-vocabulary/UserVocabularyCreationScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactElement, useState } from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
+
+import { AudioCircleIcon, PhotoCircleIcon } from '../../../assets/images'
+import Button from '../../components/Button'
+import CustomTextInput from '../../components/CustomTextInput'
+import Dropdown from '../../components/Dropdown'
+import RouteWrapper from '../../components/RouteWrapper'
+import ScreenHeading from '../../components/ScreenHeading'
+import { HintText } from '../../components/text/Hint'
+import { ARTICLES, ArticleType, BUTTONS_THEME } from '../../constants/data'
+import { getLabels } from '../../services/helpers'
+
+const Root = styled.View`
+  padding: ${props => props.theme.spacings.md};
+  flex: 1;
+`
+
+const SaveButton = styled(Button)`
+  margin-top: ${props => props.theme.spacings.lg};
+  align-self: center;
+`
+
+const AddAudioButton = styled(Button)`
+  margin-top: ${props => props.theme.spacings.md};
+  justify-content: flex-start;
+  padding: 0;
+`
+const AddPictureButton = styled(AddAudioButton)`
+  margin-bottom: 0;
+`
+
+const StyledCustomTextInput = styled(CustomTextInput)`
+  margin-bottom: ${props => props.theme.spacings.sm};
+`
+
+const StyledHintText = styled(HintText)`
+  margin-left: ${props => props.theme.spacings.xxl};
+`
+
+const UserVocabularyCreationScreen = (): ReactElement => {
+  const [word, setWord] = useState<string>('')
+  const [articleId, setArticleId] = useState<number | null>(null)
```

Comment:
Would it make sense to have one article selected as default? Maybe it would make creating own vocabulary a little bit faster if the correct one is already selected. However, the downside may be that user miss that they may have to select another article.

Reply:
hm i don't think this is a good idea. what to choose for default?

Reply:
Agree with @f1sh1918